src/index.ts

```diff
@@ -22,15 +21,20 @@ const service = () => {
     defaultService = new Service({
       token: token,
       mesgConfig: mesgConfig,
-      client: new ServiceClient(endpointTCP, grpc.credentials.createInsecure()),
+      client: new ServiceClient(defaultEndpoint, grpc.credentials.createInsecure()),
     });
   }
 
   return defaultService;
 }
 
-const application = () => {
-  if(!defaultApplication){
+type ApplicationOptions = {
+  endpoint?: string
+}
+
+const application = (options?: ApplicationOptions) => {
+  if (!defaultApplication){
+    const endpoint = options && options.endpoint ? options.endpoint: defaultEndpoint;
```

Comment:
formatting issue and clarity
```suggestion
    const endpoint = (options && options.endpoint) ? options.endpoint : defaultEndpoint;
```